Add c-rtk info

cuhome opened the pull request:
Modify c-rtk info

hamishwillee commented on en/peripherals/rtk_gps_cuav_c-rtk.md:
```diff
@@ -0,0 +1,22 @@
+
+The [CUAV C-RTK GPS receiver](http://doc.cuav.net/gps/c-rtk/en/) is a Real Time Kinematics module for mass market launched by CUAV. The complete RTK system can be composed of two C-RTK modules (one for aircraft and one for base station).Using RTK, PX4 can get its position with centimetre-level accuracy, which is much more accurate than can be provided by a normal GPS.
+
+<img src="../../images/rtk_c_rtk.jpg" />
```
Change to 
```
![C RTK](../../images/rtk_c_rtk.jpg)
```

hamishwillee commented on en/peripherals/rtk_gps_cuav_c-rtk.md:
```diff
@@ -0,0 +1,22 @@
+# CUAV C-RTK
+
+The [CUAV C-RTK GPS receiver](http://doc.cuav.net/gps/c-rtk/en/) is a Real Time Kinematics module for mass market launched by CUAV. The complete RTK system can be composed of two C-RTK modules (one for aircraft and one for base station).Using RTK, PX4 can get its position with centimetre-level accuracy, which is much more accurate than can be provided by a normal GPS.
```
Add space after full stop here: `station).Using`

hamishwillee replied:
"The complete RTK system can be composed of two C-RTK modules (one for aircraft and one for base station)" - is it a normal RTK system? Ie can it also support multiple rovers for a single base station?

hamishwillee replied:
Make "Real Time Kinematics" a link like this `[Real Time Kinematics](../advanced_features/rtk-gps.md)`

hamishwillee commented on en/peripherals/rtk_gps_cuav_c-rtk.md:
```diff
@@ -0,0 +1,22 @@
+
+## Purchase
+
+[cuav taobao](https://item.taobao.com/item.htm?id=565380634341&spm=2014.21600712.0.0)
```
Make these separate bullet points - e.g. 
```
- [cuav taobao](https://item.taobao.com/item.htm?id=565380634341&spm=2014.21600712.0.0)
- [...
```

hamishwillee commented:
Thanks for this. In addition to changes inline:
1. Please update this doc to include your RTK option! https://github.com/cuav/px4_user_guide/blob/master/en/advanced_features/rtk-gps.md#supported-rtk-devices
2. Currently this will not be rendered. Can you please also update [SUMMARY.md](https://github.com/cuav/px4_user_guide/blob/master/en/SUMMARY.md) under the Drotek entry.

ndepal commented on en/peripherals/rtk_gps_cuav_c-rtk.md:
```diff
@@ -0,0 +1,22 @@
+
+RTK setup and use on PX4 via *QGroundControl* is largely plug and play (see [RTK GPS](../advanced_features/rtk-gps.md) for more information).
+
+> **Tip** The Drotek UCenter tool is not required for configuration - everything is handled by *QGroundControl*.
```
UCenter is not made by Drotek, but by u-blox.

cuhome commented:
The above problem has been fixed and the relevant description has been modified.

hamishwillee commented on en/peripherals/rtk_gps_cuav_c-rtk.md:
```diff
@@ -0,0 +1,43 @@
+
+## Wiring and Connections
+
+C-RTK GPS with 6-pin connector and 4-pin connector, 6-pin connector can be plugged directly into [Pixhack v3](http://doc.cuav.net/gps/c-rtk/en/Instructions.html) GPS UART port, 4-pin connector can not be connected or access pixhack v3 ser4/5 port
```
> 4-pin connector can not be connected or access pixhack v3 ser4/5 port

So what is the 4 pin connector for - and what platforms can it be connected to? If it can be connected to anything probably worth listing that and doing pinout.

hamishwillee replied:
So basically I think you are saying that you connect the 6 pin connector (only) for Pixhack, Pixhawk3Pro and Pixracer. For Pixhack you can connect directly, for the Pixhawk3 Pro and Pixracer you need to modify the cable. Is that correct?

Please confirm you do not need the 4 pin connector for those other boards.

cuhome replied:
The 4pin connector can be connected to the uart4 of the px4 flight controller as gps2, but the px4 firmware has not found a way to enable the second gps.

hamishwillee commented on en/peripherals/rtk_gps_cuav_c-rtk.md:
```diff
@@ -0,0 +1,43 @@
+
+
+### Pinout
+The C-RTK GPS pinout is provided below. This can be used to help modify the connector for other autopilot boards.
```
This is excellent than you. Probably worth having the pinout for the 4 pin connector 2.

hamishwillee commented on en/peripherals/rtk_gps_cuav_c-rtk.md:
```diff
@@ -0,0 +1,43 @@
+
+The Pixhawk 3 Pro and Pixracer have 6-pin GPS port connectors. For these controllers, you can modify the order of the GPS cables \(shown below\).![C-RTK_6PIN](/assets/rtk_cuav_c-rtk_to_6pin_connector.jpg)
+
+4-pin connector for GPS2 - This cable can also be connected if desired.
```
What does this mean - what is GPS2? When would you need to use this? What does it give you? Does it require any additional on-board configuration (nothing mentioned about this in the RTK doc). All I can guess about this port so far is that it is intended to be connected to Serial 4/5 port and that it is optional.

hamishwillee commented:
Much better. A few more comments inline - mostly about the 4 pin connector.

hamishwillee commented on en/peripherals/rtk_gps_cuav_c-rtk.md:
```diff
@@ -0,0 +1,43 @@
+
+The [CUAV C-RTK GPS receiver](http://doc.cuav.net/gps/c-rtk/en/) is a RTK module for mass market launched by CUAV. The complete RTK system consists of at least two C-RTK modules \(one for the base station and the others for the aircraft\). Using RTK, PX4 can get its position with centimetre-level accuracy, which is much more accurate than can be provided by a normal GPS.
+
+![C-RTK](../../images/rtk_c-rtk.jpg)
```
I may shrink this a bit later :-)

cuhome commented:
The 4pin connector is an m8n gps interface that can be used as a second gps. But I didn't find a tutorial for using the second gps with px4 firmware. I have modified the description and added the interface definition. Please review it for me. Thank you！

hamishwillee commented:
Thanks. Looks good to me except the size of the image.  I will have some minor English corrections, but nothing for you to worry about - I will do as a post process. Plan to merge tomorrow if there are no further comments. @ndepal Anything to add. 

@bkueng Would I be correct in the assumption that PX4 only supports a single GPS?

bkueng commented:
> @bkueng Would I be correct in the assumption that PX4 only supports a single GPS?

We're working on multi-gps support. The driver can already read out 2 GPS, but the estimator does not handle them.

cuhome commented:
@hamishwillee @bkueng Thank your very much! I am a cuav engineer. I have a lot of open source hardware. I want to have further contact with your team. I hope you can contact me! Email address: hongle@cuav.net.   thanks!

hamishwillee commented:
Thanks @cuhome . This is now merged, and I have made some minor fixes in https://github.com/PX4/px4_user_guide/pull/233.

You can check this out live now: https://docs.px4.io/en/peripherals/rtk_gps_cuav_c-rtk.html

As to keeping in contact with the team, awesome! The best way to do this though is via Slack and the discussion boards: https://docs.px4.io/en/?q=#support
I can help with documents, but the whole development team is on Slack, and you'll get much better help and advice. Can't wait to see what you come up with!
